api/src/env.ts

```diff
@@ -65,7 +66,8 @@ const getJsonConfiguration = () => {
         "isDevEnvironnement": process.env.SILL_IS_DEV_ENVIRONNEMENT?.toLowerCase() === "true",
         "externalSoftwareDataOrigin": process.env.SILL_EXTERNAL_SOFTWARE_DATA_ORIGIN,
         "redirectUrl": process.env.SILL_REDIRECT_URL,
-        "databaseUrl": process.env.DATABASE_URL
+        "databaseUrl": process.env.DATABASE_URL,
+        "feedFromSource": process.env.FEEDFROMSOURCE ? process.env.FEEDFROMSOURCE.toLowerCase() == "true" : false,
```

Comment:
tu as une syntax plus courte :
```suggestion
        "feedFromSource": process.env.INITIALIZE_SOFTWARES_FROM_SOURCE?.toLowerCase() === 'true';
```